package.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "skale-oracle-demo",
+    "private": true,
+    "license": "AGPL-3.0",
+    "author": "SKALE Labs and contributors",
+    "scripts": {
+        "compile": "npx hardhat clean && npx hardhat compile",
+        "lint": "npx solhint \"contracts/**/*.sol\"",
+        "prepare": "yarn compile",
+        "deploy": "npx hardhat run migrations/deploy.ts",
```

Comment:
I think it's better to deploy production version by default and to have `deploy-test` option.

Reply:
Yes will rename